constraints/valid_values/validation_list_timestamp_ranges.isl

```diff
@@ -0,0 +1,18 @@
+type::{
```

Comment:
I know we haven't been doing this before now, but it would be nice to add a comment to each test file linking it to the part of the spec it's exercising.